src/qt/votingdialog.cpp

```diff
@@ -271,7 +271,13 @@ void VotingTableModel::resetData(bool history)
             item->vectorOfAnswers_ = iterPoll.answers;
             item->totalParticipants_ = iterPoll.total_participants;
             item->totalShares_ = iterPoll.total_shares;
-            item->url_ = QString::fromStdString(iterPoll.url);
+
+            if (iterPoll.url.find("http") == std::string::npos)
+                item->url_ = QString::fromStdString("https://" + iterPoll.url);
```

Comment:
I think we want to check that the URL starts with "http", not whether it exists anywhere within the string:
```cpp
if (iterPoll.url.find("http") != 0) ...
```
We could also left-trim whitespace from the string before checking and setting it to make the fallback more reliable, but it's not strictly necessary...

Reply:
i agree with that and idk why i did not do that.

Reply:
@iFoggz Left trim looks good. Maybe we can write a function to support all whitespace later. 

Do you want to do the check for "http" only at the beginning of the URL like I wrote above?

Reply:
What about:

```
item->url_ = QString::fromStdString(iterPoll.url).trimmed();
if(!item->url_.startsWith("https://") && !item->url_.startsWith("http://'))
   item->url_.prepend("https://");
```

Reply:
btw, are we sure that the server will support https? Maybe we should opt for http:// if no protocol is provided.

Reply:
alright will do this in a bit. and like ur suggestion. most servers forward http to http if they dont want to support http so http makes sense